middle_end/flambda/simplify/env/upwards_acc.ml

```diff
@@ -32,20 +32,22 @@ type t = {
   name_occurrences : Name_occurrences.t;
   used_closure_vars : Name_occurrences.t;
   shareable_constants : Symbol.t Static_const.Map.t;
+  used_continuation_params : Variable.Set.t;
```

Comment:
What is contained here is actually more that just the continuations parameters, it's all the variables that are marked as required by rec_uses. But we only use it for continuation parameters... So I don't know if it need any change.

Reply:
I renamed it to `required_variables` as it seems more appropriate